shared/wallets/send-form/calculate-advanced-button.tsx

```diff
@@ -42,7 +44,7 @@ const CalculateAdvancedButton = (props: CalculateAdvancedButtonProps) => {
     ) : (
       <Kb.Button
         type="Wallet"
-        label="Calculate"
+        label={hasTrivialPath ? 'Confirm details' : 'Calculate'}
```

Comment:
Trivial path sends use the same path payment RPC as all other path payments, so we first need to calculate the path. However, since we don't need to display a path, exchange rate info, etc. I decided to fake needing to calculate it with a confirm button instead.

I like this approach because otherwise there'd be no confirmation step in the send flow. Open to any other ideas on how to do this, though.